pom.xml

```diff
@@ -369,6 +369,13 @@ the License.-->
       <version>2.1.8</version>
     </dependency>
 
+    <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-devtools-v113 -->
+    <dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-devtools-v113</artifactId>
```

Comment:
why is this dependency required?

Reply:
It was the issue with my local build. Looks like the issue on the github runner is something else.